scripts/wikitables/generate_data_from_erm_model.py

```diff
@@ -18,27 +19,26 @@ def make_data(input_examples_file: str,
               output_dir: str,
               num_logical_forms: int) -> None:
     reader = WikiTablesDatasetReader(tables_directory=tables_directory,
-                                     keep_if_no_dpd=True,
+                                     keep_if_no_logical_forms=True,
                                      output_agendas=True)
     dataset = reader.read(input_examples_file)
     input_lines = []
     with open(input_examples_file) as input_file:
         input_lines = input_file.readlines()
-    # Note: Double { for escaping {.
-    new_tables_config = f"{{model: {{tables_directory: {tables_directory}}}}}"
-    archive = load_archive(archived_model_file,
-                           overrides=new_tables_config)
+    archive = load_archive(archived_model_file)
     model = archive.model
     model.training = False
     model._decoder_trainer._max_num_decoded_sequences = 100
     for instance, example_line in zip(dataset, input_lines):
         outputs = model.forward_on_instance(instance)
-        parsed_info = reader._parse_example_line(example_line)
+        instance_world = instance.fields['world'].metadata
```

Comment:
I'd have just called this `world` (you don't have `instance_logical_forms`, e.g.), but 🤷‍♂.